src/platform/forms-system/src/js/components/ContactInfoReview.jsx

```diff
@@ -48,28 +48,34 @@ const ContactInfoReview = ({ data, editPage, content, keys }) => {
   const isUS = address.addressType !== ADDRESS_TYPES.international;
   const stateOrProvince = isUS ? 'state' : 'province';
 
+  const errorMsg = errorMessage => (
+    <span className="usa-input-error-message">{errorMessage}</span>
+  );
+
   // Label: formatted value in (design) display order
   const display = [
     [content.homePhone, () => renderTelephone(homePhone)],
     [content.mobilePhone, () => renderTelephone(mobilePhone)],
+    [content.email, () => email || errorMsg(content.missingEmailError)],
```

Comment:
Do we need to worry about `email` being something like `' '`?

Reply:
Same concern with the other address fields below

Reply:
If defined, all of these values should have been validated within the profile code. But given we found this problem with an undefined (or null?) zip code, I think you're right.